### DISCUSSION TITLE
Notifier updates in 7.2.0

### DISCUSSION THREAD

**Discussion by expFlower** [DataDog/chaos-controller#574] [General]
We're in the process of updating our version of the controller from 6.1.0 and i'm comparing the differences. I can see the you've added a new section the [Notifiers configuration](https://github.com/DataDog/chaos-controller/blob/13f786e1a3c6dc4d05cf7026bd9236fabcf7f73a/docs/features.md#configuration): specifically:

``` 
datadog:
    enabled: true/false # enables the datadog notifier
```

However when i look at the [configmap](https://github.com/DataDog/chaos-controller/blob/13f786e1a3c6dc4d05cf7026bd9236fabcf7f73a/chart/templates/configmap.yaml#L19-L34) there's no mention of the new section? 😕

**Comment by ptnapoleon** [comment]
Thanks for catching this! https://github.com/DataDog/chaos-controller/pull/581
